src/Http/Http.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -5,7 +5,7 @@ abstract Microsoft.AspNetCore.Http.EndpointFilterInvocationContext.Arguments.get
 abstract Microsoft.AspNetCore.Http.EndpointFilterInvocationContext.GetArgument<T>(int index) -> T
 abstract Microsoft.AspNetCore.Http.EndpointFilterInvocationContext.HttpContext.get -> Microsoft.AspNetCore.Http.HttpContext!
 Microsoft.AspNetCore.Builder.EndpointBuilder.ApplicationServices.get -> System.IServiceProvider!
-Microsoft.AspNetCore.Builder.EndpointBuilder.ApplicationServices.set -> void
+Microsoft.AspNetCore.Builder.EndpointBuilder.ApplicationServices.init -> void
```

Comment:
This isn't approved yet: https://github.com/dotnet/aspnetcore/issues/43544

It's trivial to change this back to set if we reject the API, but I think `init` makes more sense.

Reply:
This has now been approved.